spec/pronto/golang_spec.rb

```diff
@@ -44,7 +44,7 @@ module Pronto
             expect(result[5].runner).to eq(Pronto::Golang)
 
             # golint
-            expect(result[6].path).to eq('spec/fixtures/test.git/main.go')
+            expect(result[6].path).to eq('/usr/src/app/spec/fixtures/test.git/main.go')
```

Comment:
The paths should consistently be relative.
Here the code and not the test should be adapted.